### DISCUSSION TITLE
PWA manifest.json to contain POS title

### DISCUSSION THREAD

**Discussion by prusnak** [btcpayserver/btcpayserver#4063] [Ideas / Feature Requests]
Currently the file `BTCPayServer/wwwroot/manifest.json` contains fixed `name` and `short_name`:
https://github.com/btcpayserver/btcpayserver/blob/4aa94d5a1318f07ed03367b996b2d5d06476775c/BTCPayServer/wwwroot/manifest.json#L2-L3 

This means when a user adds their POS to Android home-screen, the title of the icon will be always "BTCPay POS". This can be very confusing when a user adds more POS shortcuts and does not rename them accordingly.

It would be much nicer if manifest.json contained the `$title` as `short_name` and `BTCPayServer: $title` as `name` (where `$title` is a display title of the POS, the same value that is used as `.head.title` and `.body.h1` in the POS html).

**Comment by Kukks** [comment]
https://github.com/btcpayserver/btcpayserver/pull/4064
